CHANGELOG.md

```diff
@@ -9,6 +9,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
```

Comment:
Should we remove this section? I don't really see a use as any change merged into the master branch should trigger a new version. Or can we have versions that don't correspond to a release?

Reply:
We talked about this during the meeting.

I think you should move everything under [2.0] to unreleased, right? There is no released version 2.0 yet so that header could be removed until we make the release. Alternatively we could make a release just after merging, but only if you think that makes sense as a release point.